code/game/turfs/simulated/walls.dm

```diff
@@ -374,15 +374,32 @@
 	if(istype(I, /obj/item/gun/energy/plasmacutter))
 		to_chat(user, "<span class='notice'>You begin slicing through the outer plating.</span>")
 		playsound(src, I.usesound, 100, 1)
-
-		if(do_after(user, istype(sheet_type, /obj/item/stack/sheet/mineral/diamond) ? 120 * I.toolspeed : 60 * I.toolspeed, target = src))
```

Comment:
Unintended change about diamond walls no longer taking more time to deconstruct?

Reply:
I removed it due to being incredibly hyper specific, with diamond walls almost never made, as well as welding tools and sonic jackhammer not having this slowdown. I can add it back in, if it is prefered